lib/modules/platform/bitbucket/index.ts

```diff
@@ -894,8 +883,9 @@ export async function createPr({
         },
       )
     ).body;
+    BitbucketPrCache.init(config.repository, renovateUserUuid).addPr(prRes);
     const pr = utils.prInfo(prRes);
-    // istanbul ignore if
+    // istanbul ignore if: prList is initialized at this point
     if (config.prList) {
       config.prList.push(pr);
     }
```

Comment:
```suggestion
     config.prList!.push(pr);
```
should we use this then?

Reply:
I would like to get rid of it, as it's most probably isn't a bottleneck

Reply:
But facing similar decision for GitHub, we left it intact

Reply:
Do we use it at runtime in GitHub, or is it designed to just make the cached list "less wrong" at the end?

Reply:
Without something like `prList` (being stored ad-hoc or at the memory cache), it's harder to prevent `findPr()` from issuing at least one sync request on each call (it will call `getPrList()` under the hood).

Because of that, I came to conclusion we should keep the cache two-level, but incapsulate `prList` to the PR cache, instead of using it explicitly in the `index.ts` file.